controller.js

```diff
@@ -24,5 +24,50 @@ module.exports = {
     }).then(countries => {
       res(countries);
     });
+  },
+
+
+/**
+  * creates api log from req body by taking paramters from it
+  * @function setApi
+  * @param {String} userid The userid (primary key) to get a user.
+  * @return {json} status and an object in json format
```

Comment:
The params and return description doesn't seem to be correct

Reply:
Yes ,I will correct it accordingly.

Reply:
Updated the correct comments.